net.solarnetwork.node.setup.web/web/WEB-INF/jsp/default-head-resources.jsp

```diff
@@ -34,6 +35,7 @@
 	/js/settings.js
 	/js/new-node.js
 	/js/plugins.js
+	/js/graph-data.js
```

Comment:
I'd rename this for clarity, like `datum-charts.js` to go along with the existing `datum.js` script.